javascript/javascript3/week1/Giphy_Api/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
Hi Svetlana, Thanks for reviewing my homeworks. unfortunately i didn't know that i need to create every new branch under the 'main' so every new PR is including the previous one since i created new week's branch under previous branch so when merged the Databases week1 homework it merged all the other PRs. 
It takes default DKK inside JS script if user doesn't select any other currency but i didn't do with EUR part since i left first choice to user :)